tox.ini

```diff
@@ -53,7 +53,6 @@ deps =
     certifi<=2022.6.15.1
     google-api-python-client<=2.60.0
     idna<=3.3
-    matplotlib>=3.3.0,<3.4
```

Comment:
The matplotlib installation was failing on mac and win 3.10, I started looking into the issue but found out that we don't even use matplotlib anywhere in the tests so I removed it